en/features/graphql.md

```diff
@@ -1,22 +1,23 @@
 # API Documentation
 
-* [Characteristics](#characteristics)
-* [GraphQL](#graphql)
-* [Making API requests](#making-api-requests)
-  * [Supported clients](#supported-clients)
-    * [GraphiQL](#graphiql)
-    * [Postman](#postman)
-    * [HTTP libraries](#http-libraries)
-* [Available information](#available-information)
-* [Examples of queries](#examples-of-queries)
-  * [Request a single record from a collection](#request-a-single-record-from-a-collection)
-  * [Request a complete collection](#request-a-complete-collection)
-    * [Pagination](#pagination)
-  * [Accessing several resources in a single request](#accessing-several-resources-in-a-single-request)
-* [Security limitations](#security-limitations)
-  * [Example of too deep query](#example-of-too-deep-query)
-  * [Example of too complex query](#example-of-too-complex-query)
-* [Code examples](#code-examples)
+- [API Documentation](#api-documentation)
```

Comment:
I am not sure if we need this line 🤔 . 
If we remove it this page is still displayed the same in both English and Spanish.

Reply:
Ops, sorry. You're right, it shouldn't be there. I'll change it.